extensions/grpc/compilers/download.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script downloads `protoc` releases and unzips them locally for
+# Linux, Mac OSX, and Windows platforms.
+
+DIR=`dirname $0`
+
+PB_REL="https://github.com/protocolbuffers/protobuf/releases"
+PB_VER="3.13.0"
+
+pushd $DIR 2>&1 1>/dev/null
+
+curl -LOs $PB_REL/download/v${PB_VER}/protoc-${PB_VER}-linux-x86_64.zip
+unzip -o protoc-${PB_VER}-linux-x86_64.zip -d linux "bin/*" readme.txt
+
+curl -LOs $PB_REL/download/v${PB_VER}/protoc-${PB_VER}-osx-x86_64.zip
+unzip -o protoc-${PB_VER}-osx-x86_64.zip -d darwin "bin/*" readme.txt
+
+curl -LOs $PB_REL/download/v${PB_VER}/protoc-${PB_VER}-win64.zip
+unzip -o protoc-${PB_VER}-win64.zip -d win32 "bin/*" readme.txt
```

Comment:
How about other platforms like the upcoming "Apple Silicon" (`osx-arm64`?)

I am a bit reluctant to keep `protoc` binaries in our git. Isn't there a way how to obtain them via a npm package (a dependency we can add)?

Reply:
The whole code generation story is yet to be flushed out. I'm leaning toward to have `lb4 grpc` command to do the job instead of part of this runtime package.